src/tests.ts

```diff
@@ -163,6 +163,10 @@ if (process.env.ALLOW_CONSOLE_LOGS !== "true") {
               !args[0].includes(
                 // Styled-components 5 warning
                 "You may see this warning because you've called styled inside another component"
+              ) &&
+              !args[0].includes(
+                // Expected error thrown on purpose from test
+                "##TEST_ERROR##"
```

Comment:
Lets not expand this of individual tests; if we want to silence an error inside the test, then mock it local to the test either in the `it` block or in the scope of the file:

```
console.error = jest.mock()
```